src/main/groovy/com/benjaminsproule/swagger/gradleplugin/model/SecurityDefinitionExtension.groovy

```diff
@@ -78,7 +82,8 @@ class SecurityDefinitionExtension implements ModelValidator, Swagerable<Map<Stri
         return securityDefinitions
     }
 
-    private static SecuritySchemeDefinition getSecuritySchemeDefinitionByType(String type, JsonNode node) throws GenerateException {
+    private
+    static SecuritySchemeDefinition getSecuritySchemeDefinitionByType(String type, JsonNode node) throws GenerateException {
```

Comment:
Was this intended? Looks a little wierd?

Reply:
Good spot. Lovely IntelliJ formatting...